mimic/model/identity.py

```diff
@@ -0,0 +1,22 @@
+"""
+Models relating to identity.
+"""
+from attr import attributes, attr, validators
```

Comment:
I'm still trying to figure out how this project's process works, but I think the appropriate thing is an umbrella issue, so I filed it: https://github.com/rackerlabs/mimic/issues/241